src/core.c/CompUnit/PrecompilationRepository.pm6

```diff
@@ -357,7 +366,8 @@ class CompUnit::PrecompilationRepository::Default does CompUnit::PrecompilationR
         );
         $bc.unlink;
         self.store.store-repo-id($compiler-id, $id, :repo-id($*REPO.id));
-        self.store.unlock;
+        $bcl.close;
+        #self.store.unlock;
```

Comment:
I guess closing the file will unlock it, too, but it's probably cleaner to explicitly unlock it.

Reply:
Sure thing.